aws/config.go

```diff
@@ -120,6 +120,7 @@ import (
 	"github.com/aws/aws-sdk-go/service/personalize"
 	"github.com/aws/aws-sdk-go/service/pinpoint"
 	"github.com/aws/aws-sdk-go/service/pricing"
+	"github.com/aws/aws-sdk-go/service/prometheusservice"
```

Comment:
For adding a new service to the Terraform AWS Provider, please see the [Contributing Guide section](). In particular, this pull request should additionally include:

- In `website/docs/guides/custom-service-endpoints.html.md`: Add the service name (`prometheusservice`) in the list of customizable endpoints.
- In `infrastructure/repository/labels-service.tf`: Add the new service (`prometheusservice`) to create a repository label.
- In `.hashibot.hcl`: Add the new service to automated issue and pull request labeling. e.g.

```hcl
behavior "regexp_issue_labeler_v2" "service_labels" {
  # ... other configuration ...

  label_map = {
    # ... other services ...
    "service/prometheusservice" = [
      "aws_prometheus_",
    ],
    # ... other services ...
  }
}

behavior "pull_request_path_labeler" "service_labels"
  # ... other configuration ...

  label_map = {
    # ... other services ...
    "service/prometheusservice" = [
      "aws/internal/service/prometheus/**/*",
      "**/*_prometheus_*",
      "**/prometheus_*",
    ],
    # ... other services ...
  }
}
```

Reply:
I'll make sure to include these changes in the next revision.